### ISSUE TITLE
Message shows underneath modal

### ISSUE THREAD

**Issue by ucallQA** [lucasferreira/react-native-flash-message#59]
I would like to start off by saying that this library is amazing. 

I'm having one particular issue that the message shows underneath a modal if a modal is currently open

My current root setup looks like:

```  
  <Provider store={store}>
    <ActionSheetProvider>
      <App />
    </ActionSheetProvider>
  </Provider>
```

Where App is : 

```        
        <View style={{flex: 1}}>
          <AppNavigation
            ref={this.onNavigatorRef.bind(this)}
          />
          <FlashMessage position="top" />
        </View>
```

Any help would be appreciated, thanks!

**Comment by lucasferreira** [comment]
Hi @ucallQA 

What kind of navigation lib are you using?

**Comment by ucallQA** [comment]
Hi @lucasferreira , 
Currently we are using react-navigation although it is not the latest version, maybe that's why?

**Comment by lucasferreira** [comment]
Hi @ucallQA 

I've made a small sample at expo using react-navigation and Modals:

https://snack.expo.io/SJ7LNmIKB

As you can see when you click "Go to logout screen" a new screen are open in modal way, and the flash message keeps working.

If your setup it's different from my sample, please let me know and send me more code or prints.

**Comment by ucallQA** [comment]
I actually double checked if we were using a built in modal and we are using this library

https://github.com/react-native-community/react-native-modal

I think there  might be a priority issue with this library, is there some way I can prioritize the react-native-flash-message over this one? I use the modal nested deep within one of my react components while the react-native-flash-message is initialized and called within the root/first layer of the project so I would imagine it would take priority.

**Comment by lisa-nguyen-goldenowl** [comment]
I have the same problem as @ucallQA

**Comment by lucasferreira** [comment]
Hi @ucallQA and @lisa-nguyen-goldenowl 

As we could see in https://github.com/react-native-community/react-native-modal README:

_"Since react-native-modal is an extension of the original react native modal, it works in a similar fashion react-native original modal."_

The original react native modal it's a "100% native solution" it works in another UI thread inside the app like a portal, só it runs **always** above everything in your app.

The only solution we had in this case it's described here: https://github.com/lucasferreira/react-native-flash-message/issues/1#issuecomment-375636151

Basically you will need to put a second instance of `<FlashMessage />` component inside your modals and call the show message by the ref, ex: `this.refs.mySecondFlashMessage.showMessage({ ... })`

**Comment by lisa-nguyen-goldenowl** [comment]
It worked like a charm!!! Thanks, @lucasferreira!

**Comment by lucasferreira** [comment]
You're welcome @lisa-nguyen-goldenowl ;)

**Comment by GustavoBonfimS** [comment]
I had this problem and found this solution in the official examples. maybe it helps too

https://github.dev/lucasferreira/react-native-flash-message/tree/master/examples/FlashMessagePlayground

**Comment by sam9010** [comment]
@GustavoBonfimS  https://github.dev/lucasferreira/react-native-flash-message/tree/master/examples/FlashMessagePlayground/components/CustomModal was useful for me... tnx
